### DISCUSSION TITLE
Edgetx Companion

### DISCUSSION THREAD

**Discussion by gharris07** [EdgeTX/edgetx#1914] [Q&A]
Is there an ETA on when you can download edgetx firmware using edgetx companion? Also does anyone know if Bluetooth is enabled in edgetx 2.7.  This is for the radiomaster TX16S.  Thanks.

**Comment by pfeerick** [comment]
Unfortunately no ETA yet. TX16S by default does not have bluetooth enable yet, but if you check out https://github.com/EdgeTX/edgetx/discussions/1836 there has already been requests for TX16S bluetooth so there are builds with and without bluetooth (English language).
